state/executor.go

```diff
@@ -414,6 +423,38 @@ func (t *Transition) nonceCheck(msg *types.Transaction) error {
 	return nil
 }
 
+// checkDynamicFees checks correctness of the EIP-1559 feature-related fields.
+// Basically, makes sure gas tip cap and gas fee cap are good.
+func (t *Transition) checkDynamicFees(msg *types.Transaction) error {
```

Comment:
We could use some Unit Tests for this function if there aren't any.

Reply:
Done